sdk/postgresql/Azure.ResourceManager.PostgreSql/api/Azure.ResourceManager.PostgreSql.netstandard2.0.cs

```diff
@@ -589,12 +687,12 @@ public partial class PostgreSqlFlexibleServerData : Azure.ResourceManager.Models
         public string MinorVersion { get { throw null; } }
         public Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlFlexibleServerNetwork Network { get { throw null; } set { } }
         public System.DateTimeOffset? PointInTimeUtc { get { throw null; } set { } }
-        public int? ReplicaCapacity { get { throw null; } set { } }
+        public int? ReplicaCapacity { get { throw null; } }
```

Comment:
The setter of this property has been removed which is a breaking change.
We need to mitigate this by introduce a partial class for this type `PostgreSqlFlexibleServerData`, add the property with setter (xml document should be moved there as well) and then regenerate.
Please add a `[EditorBrowasble(EditorBrowasbleState.Never)]` attribute to the setter.